gamemodes/terrortown/gamemode/client/cl_main.lua

```diff
@@ -108,6 +108,28 @@ function GM:Initialize()
 	hook.Run("PostInitialize")
 end
 
+---
+-- Called right after the map has cleaned up (usually because game.CleanUpMap was called)
+-- @hook
+-- @realm client
+-- @ref https://wiki.facepunch.com/gmod/GM:PostCleanupMap
+-- @local
+function GM:PostCleanupMap()
+	door.SetUp()
+
+	hook.Run("TTT2PostCleanupMap")
+end
+
+---
+-- Called right after the map has been cleaned up (usually because game.CleanUpMap was called).
+-- This hook is called after the @{outputs} library is set up and map entity outputs can be
+-- registered.
+-- @hook
+-- @realm client
+function GM:TTT2PostCleanupMap()
```

Comment:
If it's shared, make it shared :D

Reply:
But it is once in the client and once in the server file. I though we discussed this and ended up with anotating it the way I did it.

Reply:
Why not moving to sh_main.lua?

Reply:
I could, yes. I left it there because it is called in this file. But there's no real reason to keep it here I suppose